docs/tempo/website/release-notes/v1-5.md

```diff
@@ -0,0 +1,110 @@
+---
+title: V1.5
+weight: 70
+---
+
+# Version 1.5 release notes
+
+The Tempo team is pleased to announce the release of Tempo 1.5.
+
+With this release, Tempo adds Parquet block support (experimental), anonymous usage reporting to help the Tempo team and Grafana understand how Tempo is used, and enhancements to the metrics-generator. 
+
+We encourage users to upgrade to Grafana 9.1 to see additional Tempo-powered capabilities,
+such as the [application performance management (APM) dashboard](https://grafana.com/docs/tempo/latest/metrics-generator/app-performance-mgmt/) which visualizes span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+
+>**Note**: For a complete list of changes and enhancements, refer to the [Tempo 1.5 changelog](https://github.com/grafana/tempo/releases). 
+
+
+## Features and enhancements
+
+Version 1.5 contains the following features and enhancements. 
+
+### Apache Parquet block format support
+
+Tempo now has a columnar block format based on Apache Parquet.
+A columnar block format may result in improved search performance and also enables a large ecosystem of tools access to the underlying trace data.
```

Comment:
```suggestion
We're excited about this new format because it will enable increased speed and efficiency for Tempo search. It also allows users to leverage the large ecosystem of existing Parquet tools and libraries to process and transform their trace data.
```

Reply:
I just felt like we were underselling the excitement here with 'may result'. I'm guessing we wouldn't have bothered to do Parquet if we only thought this was a possibility.